example/config.yml

```diff
@@ -30,6 +30,8 @@ collections: # A list of collections the CMS should be able to edit
 
   - name: "settings"
     label: "Settings"
+    editor:
+      disable_preview: true
```

Comment:
I suggest changing the setting to "editor.preview" and making it default to "true". Setting a property with "disable" in the name to "true" is an unnecessary negation of the actual value we want to know: *does this collection use previews?*